web/themes/joinup/prototype/sass/_mixins.sass

```diff
@@ -72,6 +72,87 @@
     -ms-transform: translate(0, -50%)
     transform: translate(0, -50%)
 
+// Hidden print
 =hidden-print()
   @media print
     display: none !important
+
+// Flexbox
+// Defines a block flex container; enables a flex context for all its direct children.
+=flex
+  display: -webkit-box
+  display: -webkit-flex
+  display: -moz-flex
+  display: -ms-flexbox
+  display: flex
+
+// Inline flex
+// Defines an inline flex container; enables a flex context for all its direct children.
+=inline-flex
+  display: -webkit-inline-box
```

Comment:
Hi @luke-waat changes are ok and I approved the PR. Will await the build to give results and hopefully it's time to merge this one.
Just as a side info, I don't think you need these mixins. The same result would be to include [postcss tools in gulp](https://github.com/postcss/autoprefixer#gulp) that will automatically add the prefixes based on instructions or `browserlist` file. It'll just be simpler for you, as for example, you write `display: flex` for example, and gulp does the rest in the dist for you.